style.css

```diff
@@ -0,0 +1,475 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap");
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: black;
+  color: #ffcba4;
+  padding: 0rem 1rem;
+}
+.navigations {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-left: 40px;
+  padding-top: 1rem;
+  padding-bottom: 2rem;
+}
+.nav-head {
+  font-weight: bolder;
+}
+.nav-input {
+  padding: 1rem;
+  margin-left: 30rem;
+  border: none;
+  width: 40%;
+  background-color: #15181c;
+  border-radius: 20px;
```

Comment:
you could use variables for repetitive color.

Reply:
ok i will change things you mentioned